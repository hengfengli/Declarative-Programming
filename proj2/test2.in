+---+---+---+
|...|.1.|.82|
|...|9..|5..|
|4..|.8.|9.3|
+---+---+---+
|978|..3|...|
|...|865|...|
|...|7..|814|
+---+---+---+
|1.3|.2.|..9|
|..4|..9|...|
|78.|.5.|...|
+---+---+---+
