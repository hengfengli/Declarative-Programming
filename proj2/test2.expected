+---+---+---+
|397|516|482|
|862|934|571|
|415|287|963|
+---+---+---+
|978|143|625|
|241|865|397|
|536|792|814|
+---+---+---+
|153|628|749|
|624|379|158|
|789|451|236|
+---+---+---+
